docs/counters.md

```diff
@@ -0,0 +1,455 @@
+
+#### Suggested User Actions
+
+1. Ensure that the lines in your MCF are in the form `<property>: <value>`.
+
+### MCF_MalformedNodeName
+
+Value of `Node` prop either included a comma or started with a quote.
+
+#### Suggested User Actions
+
+1. Check the error message for the specifics of the malformation.
+
+### MCF_UnexpectedProperty
+
+A regular `<property>: <value>` line was found without a preceding `Node` line to associate with.
+
+### MCF_MalformedNode
+
+Either;
+1. Found a 'Node' without properties, or
+1. The value of the `Node` property was surrounded by quotes (must be non-quoted), or
+1. The value of the `Node` property included a comma (must be a unary value).
+
+### MCF_MalformedComplexValue
+
+Found malformed [complex value](complex_values.md) without a closing bracket (`]`).
+
+### MCF_LocalReferenceInResolvedFile
+
+Found an internal `l:` reference in resolved entity value.
+
+### TMCF_MalformedEntity
+
+When parsing the first (`Node: <value>`) line of a node in TMCF, the value did
+not have the required `E:` prefix to be an entity name.
+
+### TMCF_MalformedSchemaTerm
+
+TMCF had a malformed entity/column; the value must have a `->` delimeter that was missing.
+
+### TMCF_UnsupportedColumnNameInProperty
+
+A TMCF property referencing a CSV column was found. This is not supported yet.
+
+### TMCF_TmcfEntityAsDcid
+
+In TMCF, value of DCID was an `E:` entity. However, this must instead be a `C:`
+column or a constant value.
+
+### TMCF_UnexpectedNonColumn
+
+Expected value to be a TMCF column that starts with a `C:` value, but it was not.
+
+## Resolution Counters
+These counters are logged when there are errors assigning DCIDs to each node in
+the graph.
+
+### Resolution_UnresolvedExternalId_
+
+External ID reference could not be resolved.
+
+**Suffix Description:** Property for which the ID could not be resolved.
+
+#### Suggested User Actions
+
+1. Try searching for the ID on the [Data Commons Browser](https://datacommons.org/search).
+
+### Resolution_DivergingDcidsForExternalIds_
+
+External IDs resolved to different DCIDs, however, they must all map to the same DCID.
+
+**Suffix Description:** The properties that were found, separated by an underscore `_`.
+For example, a counter named `Resolution_DivergingDcidsForExternalIds_isoCode_wikidataId`
+means that the `isoCode` and `wikidataId` properties were both external IDs, but
+they resolved to different DCIDs (which is not permitted).
+
+#### Suggested User Actions
+
+1. Try searching for the DCIDs on the [Data Commons Browser](https://datacommons.org/search) and/or in your local schema (.mcf) files, and making sure they resolve to the correct and identical entity as
+uniquely identified by its DCID.
+
+### Resolution_IrreplaceableLocalRef
+
+Unable to replace a local reference.
+
+This is likely a cycle of local references, which the import tool is not able to resolve. 
+
+#### Suggested User Actions
```

Comment:
should we do: `** Suggested User Actions: **`

(That way the sections are all topics or counters)